arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -715,6 +795,64 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/workflows/{workflowName}/versions/{versionId}/triggers/{triggerName}/listCallbackUrl": {
+      "post": {
```

Comment:
Due to RBAC the call get a callbackURL needs to be behind POST. Our API convention is to start these kind of calls with "list*" to indicate it is getting a resource using POST method.